### PR TITLE
Fixes #312

### LINKED ISSUE #312: Make message about who can add "good first issue" labels more clear
Make sure its message is clear about who can add "good first issue" labels (currently just says Sandeep but that's not true).

### PR DESCRIPTION
**Pull request by krishnarao22**
## Explanation

This is a PR that fixes #312 by looping in everyone that can add a "good first issue" label

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

Note: Have had issues manually deploying and testing on oppiabot

### CODE REVIEW

**Comment by krishnarao22** [comment]
**Key concern: pinging people too much as a result of this fix?**

**Comment by U8NWXD** [comment]
What if we ping the onboarding lead and then list (without at-mentioning) the other users who can add the label? Something like: `I am removing the label for now and looping in @onboardingLead to approve the label. onboardingMember1, onboardingMember2, ... can also approve the label.`

**Comment by krishnarao22** [comment]
Hey @U8NWXD, sorry for the delay. As mentioned in #312, Sean created a new team for good first issue labelers. I changed the comment body to ping them. PTAL when you can, thanks!

**Review by U8NWXD — APPROVED** [review]

**Comment by U8NWXD** [comment]
LGTM. CI checks are failing though

**Comment by vojtechjelinek** [comment]
@krishnarao22 Can you please fix the frontend tests?

**Comment by vojtechjelinek** [comment]
@krishnarao22 I've fixed the frontend tests as we need to release this pretty soon.

File: actions/src/issues/checkIssueLabels.js [hunk 1]
```diff
@@ -50,12 +50,14 @@ const checkLabels = async () => {
 const handleGoodFirstIssueLabel = async (octokit, user) => {
   const issueNumber = context.payload.issue.number;
   // Comment on the issue and ping the onboarding team lead.
+  var commentBody = (
+    'Hi @' + user + ', only certain users are allowed to add good ' +
+    'first issue labels. Looping in @oppia/oppia-good-first-issue-labelers ' + 
+    'to add the label.'
```
Comment (seanlip) [inline actions/src/issues/checkIssueLabels.js:56]: Maybe add "Thanks!" at the end so the developer doesn't feel they did something wrong.

**Review by seanlip — APPROVED** [review]
Thanks @vojtechjelinek! Just one suggestion.

**Comment by krishnarao22** [comment]
Thanks @vojtechjelinek -- was having a bit of trouble with this. Going to merge this PR.

**Pull request merged**